Sources/DittoDiskUsage/DittoDiskUsageView.swift

```diff
@@ -8,13 +8,20 @@
 import DittoSwift
 import SwiftUI
 import Combine
+import DittoExportData
 
 public struct DittoDiskUsageView: View {
 
     @Environment(\.presentationMode) var presentationMode
+    @Environment(\.colorScheme) private var colorScheme
 
+    // Export Ditto Directory
+    @State private var presentExportDataShare: Bool = false
+    @State private var presentExportDataAlert: Bool = false
+    private var textColor: Color {
+        colorScheme == .dark ? Color(UIColor.systemBlue) : .blue
```

Comment:
I think the `UIColor` usage here might need to be gated around some check for UIKit availability - does this build for tvOS?

Reply:
I just removed that property all together as it was pulled in from other code and is not needed here.